File: material-support/app/src/main/java/me/cafecode/android/material/MainActivity.java
package me.cafecode.android.material;

import android.content.Intent;
import android.support.v7.app.ActionBarActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ListView;


public class MainActivity extends ActionBarActivity implements AdapterView.OnItemClickListener {

    private ListView mList;
    String[] menu = {"Typography", "Buttons"};
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mList = (ListView) findViewById(R.id.list);
        mList.setAdapter(new ArrayAdapter(this, android.R.layout.simple_list_item_1, android.R.id.text1, menu));
        mList.setOnItemClickListener(this);
    }

    @Override
    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
        if (position == 0) {
            startActivity(new Intent(this, TypographyActivity.class));
        } else if (position == 1) {
            startActivity(new Intent(this, ButtonsActivity.class));
        }
    }
}

File: qr-code/app/src/main/java/com/example/nux/qr_code/MainActivity.kt
package com.example.nux.qr_code

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        scanButton.setOnClickListener { startActivity(Intent(this@MainActivity, QrCodeScannerActivity::class.java)) }
    }
}
